src/components/Department/Department.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import {
+  Button,
+  ButtonGroup,
+  Col,
+  Container,
+  Row,
+} from 'reactstrap';
+import API from '../../API';
+import './Department.css';
+
+export default class Department extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      departmentInfo: null,
+      error: null,
+    };
+  }
+
+  componentDidMount() {
+    const getDepartmentInfo = async () => {
+      const { match: { params: { id } } } = this.props || null;
+      const data = await API.getDepartmentInfo(id);
+
+      if (data.status !== 200) {
+        this.setState({ error: data });
+      } else {
+        this.setState({ departmentInfo: data.data.description });
+      }
```

Comment:
don't rely on the status code there. 
Potential alternatives
const {data, error} = request
or try catch at least